javascript/javascript3/week1/WeatherApp/main.js

```diff
@@ -0,0 +1,130 @@
+const btn1 = document.getElementById('btnGetCity');
+const btn2 = document.getElementById('btnGetLoc');
```

Comment:
btn1 and btn2 could have slightly more descriptive names - for instance btnGetLoc and btnGetCity 😉